src/ddmd/objc.d

```diff
@@ -117,126 +117,140 @@ struct ObjcSelector
     }
 }
 
-struct Objc_ClassDeclaration
-{
-    // true if this is an Objective-C class/interface
-    bool objc;
+private __gshared Objc _objc;
 
-    // MARK: Objc_ClassDeclaration
-    extern (C++) bool isInterface()
-    {
-        return objc;
-    }
+Objc objc()
+{
+    return _objc;
 }
 
-struct Objc_FuncDeclaration
+interface Objc
```

Comment:
However looking at the travisci builds, it looks like it's only gdc that fails.  And pretty much all autotester builds.  So maybe we're triggering some lament C++ class bug in the frontend that has since fixed (I can't reproduce with gdc master).  I think the autotester uses 2.067.